sceptre/cli.py

```diff
@@ -574,6 +576,170 @@ def get_stack_policy(ctx, environment, stack):
     write(response.get('StackPolicyBody', {}))
 
 
+@cli.group(name="init")
+def init():
+    """
+    Commands for initialising Sceptre projects.
+    """
+    pass
+
+
+@init.command("env")
+@click.argument('env')
+@catch_exceptions
+@click.pass_context
+def init_environment(ctx, env):
+    """
+    Initialises an environment in a project.
+
+    Creates ENVIRONMENT folder in the project.
+    """
+    cwd = ctx.obj["sceptre_dir"]
+    for item in os.listdir(cwd):
+        # If already a config folder create a sub environment
+        if os.path.isdir(item) and item == "config":
+            config_dir = os.path.join(os.getcwd(), "config")
+            _create_new_environment(config_dir, env)
+
+
+@init.command("project")
+@catch_exceptions
+@click.argument('project_name')
+@click.pass_context
+def init_project(ctx, project_name):
+    """
+    Initialises a new project.
+
+    Creates PROJECT_NAME project folder.
+    """
+    cwd = os.getcwd()
+    sceptre_folders = {"config", "templates"}
+    project_folder = os.path.join(cwd, project_name)
+    try:
+        os.mkdir(project_folder)
+    except OSError as e:
```

Comment:
If you're only checking to see whether the directory exists and re-raising the exception otherwise, would it be just as safe and a bit more concise to just check to see whether the directory already exists?

Reply:
True, I was following the pythonic "its better to ask for forgiveness than permission" mentality.

Reply:
After thinking about it, it makes the possibility of failure more explicit.